src/synapse/workload.py

```diff
@@ -100,6 +101,20 @@ def check_alive() -> ops.pebble.CheckDict:
     return check.to_dict()
 
 
+def check_nginx_ready() -> typing.Dict:
+    """Return the Synapse NGINX container check.
+
+    Returns:
+        Dict: check object converted to its dict representation.
+    """
+    check = Check(CHECK_READY_NAME)
+    check.override = "replace"
+    check.level = "ready"
+    check.tcp = {"port": SYNAPSE_NGINX_PORT}
+    # _CheckDict cannot be imported
```

Comment:
I seem to remember that this is not right anymore?

Reply:
Oops. Removed.